src/keycodes/index.js

```diff
@@ -1,47 +1,86 @@
-import aliases from './aliases'
 import codes from './codes'
+import { splitCombination, returnCharCode } from '../helpers'
 
 const noop = () => {}
 
-const getKeyMap = (keymap, alias) => Object.keys(keymap).map(input => {
-  const result = {}
-  const { keyup, keydown } = keymap[input]
-  input.replace('numpad +', 'numpad add').split('+').forEach(keyName => {
-    switch (keyName.toLowerCase()) {
-      case 'ctrl':
-      case 'alt':
-      case 'shift':
-      case 'meta':
-        result[keyName] = true
-        break
-      default:
-        result.keyCode = alias[keyName] || searchKeyCode(keyName)
+const defaultModifiers = {
+  ctrlKey: false,
+  altKey: false,
+  shiftKey: false,
+  metaKey: false
+}
+
+const alternativeKeyNames = {
+  option: 'alt',
+  command: 'meta',
+  return: 'enter',
+  escape: 'esc',
+  plus: '+',
+  mod: /Mac|iPod|iPhone|iPad/.test(navigator.platform) ? 'meta' : 'ctrl'
```

Comment:
I think meta and Ctrl keys should be in line with the browser's definition of it

Reply:
This was actually inspired by [moustrapjs](https://github.com/ccampbell/mousetrap/), which cover almost every use case. 

https://github.com/ccampbell/mousetrap/blob/master/mousetrap.js#L123-L136